ocaml/idl/datamodel_pool.ml

```diff
@@ -75,6 +75,21 @@ open Datamodel_types
       ~allowed_roles:_R_POOL_OP
       ()
 
+  (* This is a map of uuid -> cert_blob *)
+  let pool_certs = Map (String, String)
+
+  let pre_join_exchange_certificates = call
```

Comment:
I find it a bit strange that some of the cert distrib is done here, and the rest is done in cert_distrib.ml. I would prefer if it were all done in cert_distrib.ml (i think the way to do that would be to add command to `WireProtocol.command`), or alternatively if all the commands in cert_distrib.ml were made part of the datamodel

Reply:
Linguistically this is a bit awkward. Should this be a "pre-join certificates exchange" or a "certificates pre-join check"?

Reply:
I think I may be able to avoid changing the datamodel by reusing the untrusted connection that the joiner creates to direct the function call of `cert_distrib_atom` to the pool.

Reply:
Done!